cosmic-usage-core/src/main/java/com/github/missioncriticalcloud/cosmic/usage/core/services/impl/TokenServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+package com.github.missioncriticalcloud.cosmic.usage.core.services.impl;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+import com.github.missioncriticalcloud.cosmic.usage.core.model.DomainToken;
+import com.github.missioncriticalcloud.cosmic.usage.core.services.TokenService;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.crypto.encrypt.Encryptors;
+import org.springframework.security.crypto.encrypt.TextEncryptor;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TokenServiceImpl implements TokenService {
+
+    private final TextEncryptor encryptor;
+    private final ObjectMapper objectMapper;
+
+    private final long ttl;
+
+    private final DateTimeZone dateTimeZone = DateTimeZone.UTC;
```

Comment:
In my opinion the environment should be responsible for defining the timezone, not the app.